### ISSUE TITLE
Add Verena's Kaggle exploration notebook to the monorepo

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/sage-monorepo#1222]
See #1221 for background information.

**Comment by vpchung** [comment]
I am unable to successfully launch Jupyter notebook:

![Image](https://user-images.githubusercontent.com/9377970/219507933-8b02f2bd-6f80-4a67-ab03-5b6b5e99e22c.png)

Following the Preview in #1221 , the commands I used were:

```
$ nx prepare-python openchallenges-notebook
$ nx serve openchallenges-notebook
```

I initially thought it was maybe due to the recent change to the config server, so I tried adding `PORT=8888` to my `.env` but to no avail.  Then I looked up the `Fail to get yarn configuration. Unknown Syntax Error: Extraneous positional argument ("list").` error message and came across this [issued bug](https://github.com/jupyterlab/jupyterlab/issues/12848) from July 2022.  Based on some of the comments, yarn 3 is not yet supported and we are currently using `v3.3.0`. 

@tschaffter did you happen to come across this error back when you first added the notebook a month ago?
